### PR TITLE
[15.0] [mig] web_notify

### PR DESCRIPTION
**Pull request by aiendry-aktivsoftware**

### CODE REVIEW

File: web_notify/static/src/js/services/notification_services.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+odoo.define("web_notify.NotificationService", function (require) {
+    "use strict";
+    const {browser} = require("@web/core/browser/browser");
+    const {registry} = require("@web/core/registry");
+
+    const webNotificationService = {
```
Comment (chienandalu) [inline web_notify/static/src/js/services/notification_services.js:6]:
```suggestion
/** @odoo-module **/

import { browser } from "@web/core/browser/browser";
import { registry } from "@web/core/registry";

export const webNotificationService = {
```

File: web_notify/static/src/js/services/notification.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+odoo.define("web_notify.Notification", function (require) {
+    "use strict";
+    const {Notification} = require("@web/core/notifications/notification");
+    const {patch} = require("web.utils");
```
Comment (chienandalu) [inline web_notify/static/src/js/services/notification.js:4]:
```suggestion
/** @odoo-module */

import {Notification} from "@web/core/notifications/notification";
import { patch } from "web.utils";

**Review by chienandalu — CHANGES_REQUESTED** [review]
Functionally ok. Thanks!

Can you use ES6 completly (suggestions below; you'll also have to rename the js files to whatever_file.esm.js) so it's not necessary to do it in latter migrations?

**Review by CarlosRoca13 — APPROVED** [review]
Thanks :+1: 

Please review the comments of @chienandalu

**Review by chienandalu — APPROVED** [review]
Non blocking this

**Comment by pedrobaeza** [comment]
@chienandalu I think you should supersede the PR respecting his attribution, but adding your improvements.

**Comment by pedrobaeza** [comment]
Superseded by #2381

**Pull request closed without merging**